doc/man3/OSSL_LIB_CTX.pod

```diff
@@ -111,6 +113,11 @@ in the mean time.  This means that the calling thread must not free the
 library context that was the default at the start of the async job before
 that job has finished.
 
+OSSL_LIB_CTX_get_data() returns address represeing desired item kept in
```

Comment:
returns **an** address **representing** **the** desired item kept in **the** library context.